src/components/Popover/PopoverExample.tsx

```diff
@@ -0,0 +1,129 @@
+import { PopoverHeader } from '../PopoverHeader/PopoverHeader';
+import { TextPassage } from '../TextPassage/TextPassage';
+
+export interface Props {
+  /**
+   * Child node(s) that can be nested inside component
+   */
+  children: ReactNode;
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const PopoverExample: React.FC<Props> = ({
+  children,
+  className,
+  ...other
+}) => {
+  const [popoverOpen, setPopoverOpen] = useState(false);
+  const popoverButton = useRef() as MutableRefObject<HTMLSpanElement>;
+
+  function openContinuePopover() {
+    setPopoverOpen(true);
+  }
+
+  function closeContinuePopover(event: any) {
+    setTimeout(() => {
+      popoverButton.current.focus();
+    }, 1);
+    if (event) {
+      event.preventDefault();
+    }
+    setPopoverOpen(false);
+  }
+  const componentClassName = clsx('tooltip-popover', className, {});
+  return (
+    <div
+      style={{
+        padding: '1rem',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'flex-end',
+        height: '100vh',
+        width: '180px',
+      }}
+      className={componentClassName}
+      {...other}
+    >
+      <Button text="Open Popover" onClick={openContinuePopover} />
+
+      <Popover
+        dismissible={true}
+        isActive={popoverOpen}
+        onClose={closeContinuePopover}
+        ariaLabelledBy="popover-heading-1"
+        ariaDescribedBy="popover-description-1"
+      >
+        <PopoverHeader
+          titleAfter={<Button text="Mark All Seen" size="sm" variant="bare" />}
+        >
+          <Heading id="popover-heading-1" as="h6">
+            Notifications (4)
+          </Heading>
+        </PopoverHeader>
+        <PopoverBody>
+          <NotificationList>
+            <NotificationListItem
+              href="#"
+              title="English Teacher gave you feedback"
+              date="now"
+              source="Outsiders on Trial: Self Awareness = Trial Brief Outline"
+            ></NotificationListItem>
+            <NotificationListItem
+              href="#"
+              title="English Teacher gave you feedback"
+              date="now"
+              source="Outsiders on Trial: Self Awareness = Trial Brief Outline"
+            ></NotificationListItem>
+            <NotificationListItem
+              href="#"
+              title="English Teacher gave you feedback"
+              date="now"
+              source="Outsiders on Trial: Self Awareness = Trial Brief Outline"
+            ></NotificationListItem>
+            <NotificationListItem
+              href="#"
+              title="English Teacher gave you feedback"
+              date="now"
+              source="Outsiders on Trial: Self Awareness = Trial Brief Outline"
+            ></NotificationListItem>
+          </NotificationList>
+          <PopoverHeader>
```

Comment:
Let's make this a different element. This is where we should probably have a `NotificationPopover` recipe so we can add some extra styles that aren't needed in the system

Reply:
My reasoning on it was that the Notifications Popover has two sections that are for all purposes identical; they each hold a group of notifications. So each section would use the same header. No?